src/resources/js/pages/Users.tsx

```diff
@@ -1,5 +1,84 @@
+import { Box, FormControl, InputLabel, MenuItem } from '@mui/material';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { User } from 'bmlt-root-server-client';
+import { useEffect, useState } from 'react';
+
+import RootServerApi from '../RootServerApi';
 import { strings } from '../localization';
 
 export const Users = () => {
-  return <div>{strings.usersTitle}</div>;
+  const [currentSelection, setCurrentSelection] = useState('-1');
```

Comment:
Should you be passing the number `-1` instead of the string `'-1'`?

Reply:
The MenuItem value wants a string, so I just convert the ID to a string